docs/framework/migration-guide/mitigation-serialization-control-characters.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: "Mitigation: Serialization of Control Characters with the DataContractJsonSerializer"
-ms.date: "04/07/2017"
+title: "Mitigation: Serialize control characters - DataContractJsonSerializer"
```

Comment:
I don't think this change is correct.

Reply:
How long can titles be? Maybe:

```suggestion
title: "Serialization of control characters with DataContractJsonSerializer"
```

Reply:
Or just "Serialize control characters with DataContractJsonSerializer"?

Reply:
I just find the active verb here wrong. This is about the behavior with that control.